### ISSUE TITLE
RSS Sitemap - Add documentation on NGINX rewrite rule

### ISSUE THREAD

**Issue by wpsmort** [semperfiwebdesign/all-in-one-seo-pack#3128]
Reported here - https://secure.helpscout.net/conversation/1078056140/624572?folderId=3532980

We need to determine what rule is needed for NGINX servers for our RSS sitemap and add documentation on the site.

**Comment by arnaudbroes** [comment]
@wpsmort I'm guessing this is similar to https://github.com/awesomemotive/aioseop-pro/issues/590?

**Comment by wpsmort** [comment]
@arnaudbroes Not really.  In the case of the video sitemap there are rules in place but they may not work.  With regards to the RSS sitemap, I don't believe there are any rules, we may have never added them when we added that feature.

**Comment by EkoJR** [comment]
@wpsmort RSS seems to be working on NGINX with no problems, but none-the-less here are some rules I believe would work if the problem arises.

```
# AIOSEOP Sitemap
rewrite ^/sitemap.rss$ /index.php?aiosp_sitemap_path=rss last;
rewrite ^/sitemap.latest.rss$ /index.php?aiosp_sitemap_path=rss_latest last;

# AIOSEOP Video Sitemap
rewrite ^/video-sitemap.rss$ /index.php?aiosp_video_sitemap_path=rss last;
rewrite ^/video-sitemap.latest.rss$ /index.php?aiosp_video_sitemap_path=rss_latest last;
```
